test/helpers/restricted-nori.ts

```diff
@@ -0,0 +1,84 @@
+import type { BigNumber } from 'ethers';
+
+import type { RestrictedNORI } from '@/typechain-types/contracts/RestrictedNORI';
+import type { setupTest } from '@/test/helpers';
+import { expect } from '@/test/helpers';
+
+export const SECONDS_IN_1_YEAR_AVG = 31_556_952;
```

Comment:
any chance this will break the test suite on leap years?

Reply:
shouldn't.... none of the time used in this contract or tests actually involves real dates, it's all just dealing with seconds in unix epoch.  this is just used to represent exactly 1/10th of a standard schedule duration for easy math and stuff